code/game/objects/machinery/overwatch.dm

```diff
@@ -98,7 +98,7 @@ GLOBAL_LIST_EMPTY(active_cas_targets)
 	send_defend_order = new
 	send_retreat_order = new
 	send_rally_order = new
-	cic_mini = new
+	cic_mini = (faction == FACTION_SOM ? new /datum/action/minimap/marine/external/som : new /datum/action/minimap/marine/external)
```

Comment:
This is kinda ugly, is there a cleaner fix than a faction_to_minimap_action glob?